wdfn-server/waterdata/camera.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+
+# proof of concept. should call service to get
+
+JSON_CAMERA_FILE = os.getcwd()+"/data/monitoring_camera_data.json"
+
+
+def get_json_camera_data():
+    with open(JSON_CAMERA_FILE, 'r') as json_file:
+        json_raw = json_file.read() 
+    return json_raw
+
+
+def json_raw_to_dictionary(json_raw):
+    return json.loads(json_raw)
+
+
+def get_site_camera_data(site_no):
+    json_raw = get_json_camera_data()
+    camera = json_raw_to_dictionary(json_raw)
+    return find_site_in_cameras(site_no, camera)
+
+
+def find_site_in_cameras(site_no, cameras):
```

Comment:
```suggestion
def find_site_in_cameras(site_no, cameras):
    site_camera = next((sc for sc in cameras if sc['SiteId'] == site_no), {})
    return site_camera
```

Reply:
I tried replacing this and the unit tests fail.

Reply:
Okay.